torch/distributed/distributed_c10d.py

```diff
@@ -255,10 +255,9 @@ def __init__(self, backend: Union[str, Backend]):
         if backend == Backend.UNDEFINED:
             # default config when backend is not specified
             # supported since PyTorch 2.0
-            self.device_backend_map = {
-                "cpu": Backend.GLOO,
-                "cuda": Backend.NCCL,
-            }
+            self.device_backend_map["cpu"] = Backend.GLOO
```

Comment:
```suggestion
            self.device_backend_map = {"cpu"] : Backend.GLOO}
```
is marginally faster